apps/cowswap-frontend/src/modules/permit/index.ts

```diff
@@ -0,0 +1,2 @@
+export { useIsTokenPermittable, usePermitHookData, usePermitHookParams } from './hooks'
```

Comment:
The interface of this module is great (u kept it pretty easy to use and ecapsulated the complexity inside)

However, do we need `usePermitHookParams` can't we use just `usePermitHookData` which receives the sellToken? 
Also, i would expect to find here the hook that lets me sign

Reply:
> However, do we need usePermitHookParams can't we use just usePermitHookData which receives the sellToken?

Will work on that

> Also, i would expect to find here the hook that lets me sign

It's on `modules/permit/utils/generatePermitHook`, which is true I did not export.